test/views/SummaryAnswerPageViewSpec.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import views.behaviours.ViewBehaviours
+import views.html.SummaryAnswerPageView
+
+class SummaryAnswerPageViewSpec extends ViewBehaviours {
+  val index = 0
+
+  "SummaryAnswerPage view" must {
+
+    val view = viewFor[SummaryAnswerPageView](Some(emptyUserAnswers))
+
+    val applyView = view.apply(Nil)(fakeRequest, messages)
+
+    behave like normalPage(applyView, "summaryAnswerPage")
```

Comment:
Same comment as above to have a conversation with @JohnWhitfield

Reply:
Done